### PR TITLE
Creating a data model for Mapoteca

### PR DESCRIPTION
**Pull request by smartsena**

### CODE REVIEW

File: documental/new_models.py [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+
+    def __str__(self) -> str:
+        """Returns `documental.DocsDocumentTI` string data.
+        Returns:
+            str: model data document name.
+        """
+        return self.no_document
+
+
+from django.contrib.postgres.fields.jsonb import JSONField
```
Comment (dagnaldo) [inline documental/new_models.py:276]: Why is it here?

File: documental/new_models.py [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+        """"Meta class for `documental.UsersCMR` model."""
+        app_label = 'documental'
+        verbose_name = 'User CMR'
+        verbose_name_plural = 'Users CMR'
+        # usuario = models.ForeignKey(settings.AUTH_USER_MODEL, related_name=_('usuario'))
+        # db_table = 'painel\".\"auth_user'
+        # managed = False
+
+
+class auth_institution_temp(models.Model):
```
Comment (dagnaldo) [inline documental/new_models.py:72]: what is it?

**Review by dagnaldo — CHANGES_REQUESTED** [review]

File: documental/new_models.py (hunk 1 above)
Comment (smartsena) [reply documental/new_models.py:276]: Adjusted.

File: documental/new_models.py (hunk 2 above)
Comment (smartsena) [reply documental/new_models.py:72]: Adjusted.

**Pull request merged**